nana/modules/devs.py

```diff
@@ -70,35 +69,86 @@ async def pic(chat, photo, caption=None):
     await app.send_photo(chat, photo, caption)
 
 
-async def aexec(client, message, code):
-    # Make an async function with the code and `exec` it
+async def aexec(code, client, message):
     exec(
-        'async def __ex(client, message): ' +
+        f'async def __aexec(client, message): ' +
```

Comment:
Codacy found an issue: [f-string is missing placeholders (F541)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330175)